src/melee/pl/player.h

```diff
@@ -62,16 +71,18 @@ typedef struct _StaticPlayer {
 
     /*0x68 - 0x6C*/ s32 falls[2]; /// other index for nana falls
 
-    /*0x70-0x84*/ s32 kos_by_player[6];
+    /*0x70-0x84*/ uint kos_by_player[6];
```

Comment:
`u32` will not match but this is used in unsigned comparisons. In my opinion, further proof that HAL's default data type is `int` and not `long`.